project/index.mjs

```diff
@@ -267,107 +261,99 @@ async function createNewProject(req, res) {
     res.status(201).json(logicResult)
     return
   } else {
-    utils.respondWithError(res, logicResult.status, logicResult.message)
+    respondWithError(res, logicResult.status, logicResult.message)
     return
   }
 }
 
-router
-  .route("/create")
-  .post(async (req, res, next) => {
-    // TODO: Add authentication to this endpoint
-    if (!utils.isValidJSON(req.body)) {
-      utils.respondWithError(res, 400, "Improperly formatted JSON")
-      return
-    }
-    await createNewProject(req, res)
-  })
-  .all((req, res, next) => {
```

Comment:
This `all()` may still be appropriate here. That would change the test above as well, fwiw